### ISSUE TITLE
Problem using input element with list attribute

### ISSUE THREAD

**Issue by jkleiser** [yysun/apprun#106]
In a SPA, I wanted to use code like this:
```
<form><input id="color" list="colors" /></form>
<datalist id="colors">
  <option>Red</option>
  <option>Green</option>
  <option>Blue</option>
</datalist>
```
It compiles, but in the browser console I get "TypeError: Attempted to assign to readonly property".
I'm using apprun 1.23.13 and typescript 3.9.7.
My code example is taken from https://html.com/attributes/input-list/ .

**Comment by yysun** [comment]
I will take a look

**Comment by yysun** [comment]
It is fixed ib 1.24.0 and 2.24.0
https://apprun.js.org/#play/25
